src/v2/Apps/Consign/consignRoutes.tsx

```diff
@@ -110,6 +120,13 @@ const prepareSubmissionFlowStepVariables = data => {
 }
 
 export const consignRoutes: AppRouteConfig[] = [
+  {
+    path: "/sell2",
```

Comment:
we will change the path to `/sell` later, is that correct? (since it is `/consign` right now)

Reply:
Yes, we will remove `/consign` route, change the path from `/sell2` to `/sell` and finally update links from `/consign` to `/sell`. Also, I think it makes sense to add a redirect from `/consign/*` to `/sell` just in case.

I used `/sell2` route for now because found some strange behaviour of `/sell` route. If you open https://staging.artsy.net/sell you will be redirected to https://staging.artsy.net/consign?utm_source=cadence13&utm_medium=p-podcast&utm_campaign=consignments-podcasting&utm_content=acq-ctxt&utm_term=s1, the problem here is that I don't know where the redirect happens, so I decided to use `/sell2` route, for now, to keep it working.